doc/articles/wpf-migration.md

```diff
@@ -0,0 +1,60 @@
+# Migrating WPF Apps to Web
+
+This article covers architecture and implementation considerations for migrating WPF applications to the Web using Uno Platform and WinUI.
+
+![A WPF app migrated to the web with Uno platform](Assets\migrated-app.png)
+
+## Introduction
+
+While improvements will continue to be made by Microsoft at the framework level to ensure the compatibility and reliability of your WPF application, it may be desirable to expand its reach. In this hybrid work era, previous assumptions and development decisions which served your customers well become less valid. For instance, applications targeting the Windows platform alone miss out on mobility benefits of the Web. 
+
+Advancements such as [WebAssembly (WASM)](https://developer.mozilla.org/docs/WebAssembly), a low-level bytecode for the web, have already become a viable, performant way to deliver your modern .NET applications.
+
+## Migrating the codebase
+
+The Uno Platform not only allows you to maximize reuse of C#, XAML, and .NET investments, but also enables your team to naturally leverage their existing skillset. Alongside your existing investments, you may decide to further enhance your app, refreshing aspects to make use of new capabilities in the platform. Regardless, your goal is to consider what role your application will play several years or more into the future. Depending on your scenario, it may make sense to enhance the user interface layout for different form factors. In this article, the primary focus will be on rewriting as little as possible.
+
+> [!TIP]
+> Read the UWP/WinUI & Uno documentation first to better understand the capabilities of the platform. 
+
+## Limitations
+
+The vast majority of your codebase will simply "move across". Existing XAML structure, business logic, and view models can simply be reused. You'll need to recompile old code, and won't have access to framework-only APIs. You may encounter APIs that aren't implemented or controls where a subset of the functionality is presently unsupported. Additionally, the following contain substantial enough differences from WPF to require extra research upfront:
+* [Navigation](https://docs.microsoft.com/en-us/windows/apps/design/basics/navigate-between-two-pages)
+* [Advanced binding functionality with x:Bind](https://docs.microsoft.com/en-us/windows/uwp/xaml-platform/x-bind-markup-extension)
+* [Modern APIs are often async only](https://docs.microsoft.com/en-us/windows/uwp/threading-async/call-asynchronous-apis-in-csharp-or-visual-basic)
+* [Uno has a streamlined set of XAML capabilities for performance reasons](https://docs.microsoft.com/en-us/windows/uwp/xaml-platform/xaml-overview)
+* Specific framework capabilities
```

Comment:
@davidjohnoliver I noticed @robloo published a pretty comprehensive comparison between UWP and WPF. Depending on how @robloo feels about it, maybe we want to link to that [resource](https://github.com/robloo/PublicDocs/blob/master/UWPvsWPF.md) or incorporate it here in the future?

Definitely not trying to be pushy, but what was published there is pretty incredible and would likely be a valuable addition down the road.

Reply:
I'm glad you find it useful! It's licensed CC BY-SA 4.0 so it can be used almost anywhere with attribution. The purpose is to help anyone so I would be happy to see it included in Uno here. Uno is pretty important on my end too and happy to contribute where I can.

Still hoping they take the Segoe Fluent replacement font as well :)